### DISCUSSION TITLE
Törendeki yarışmacıların sıra sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#190] [Zeka Sorusu]
Duygu,
Burak,
Funda,
Irem,
Ayşegül
Bir basketbol ödül töreni için önden arkaya doğru sıralanmıştır. Oyuncuların dizilişi şu koşulları sağlamaktadır.

- Duygu ile Burak arasında bir oyuncu vardır
- Irem Duygudan daha öndedir.
- Burak Fundadan daha öndedir ve arka arkaya değildirler.
- Ayşegül ve burak arka arkaya değildir.

Buna göre tam ortadaki oyuncu kimdir?

**Comment by Sevvalgocgun1** [comment]
Sıralama :
1-Burak
2- İrem
3-Duygu
4-Funda
5-Aysegul
